tests/src/rules/newline-after-import.js

```diff
@@ -202,7 +203,7 @@ ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
       options: [{ count: 4, exactCount: true }],
     },
     {
-      code: `var foo = require('foo-module');\n\n\n\n// Some random comment\nvar foo = 'bar';`,
+      code: `var foo = require('foo-module');\n\n\n\n\n// Some random comment\nvar foo = 'bar';`,
```

Comment:
the "count" is set to 4, so it's surprising to me that this would have 5 newlines. can you explain?

Reply:
If we print code above, it will look like this:
```javascript
var foo = require('foo-module');




// Some random comment
var foo = 'bar';
```
In other words, five newlines are needed to create four valid blank lines for the option.

Reply:
aha, thanks, that makes sende